examples/darwin-framework-tool/commands/interactive/InteractiveCommands.mm

```diff
@@ -333,6 +357,7 @@ el_status_t ExitFunction()
     el_bind_key(CTL('^'), RestartFunction);
     el_bind_key(CTL('_'), StopFunction);
     el_bind_key(CTL('@'), ExitFunction);
+    el_bind_key(CTL('Z'), SuspendOrResumeFunction);
```

Comment:
This will make it impossible to use Ctrl-Z to suspect the tool itself and get back to the shell...  Can we use a different char here, possibly?

Reply:
I’m curious to know if `Ctrl-Z` works for you in interactive mode. By that, I mean does it suspend the interactive mode and return to the shell? It doesn’t work for me, and I assumed this was because `Ctrl-Z` is already handled by `editline`. I understand that it’s currently bound to `el_ring_bell`, which is why replacing it doesn’t cause any issues.
But maybe it works for others ?